modules/cnf-topology-aware-lifecycle-manager-precache-image-filter.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// Epic CNF-6848 (4.13), Story TELCODOCS-949
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: CONCEPT
```

Comment:
This module is a bit on the edge of being a procedure (since it provides a command and output), and might need to follow the procedure module format. I'll leave the decision up to you.